homeassistant/components/tessie/sensor.py

```diff
@@ -41,6 +41,11 @@ class TessieSensorEntityDescription(SensorEntityDescription):
 
 
 DESCRIPTIONS: tuple[TessieSensorEntityDescription, ...] = (
+    TessieSensorEntityDescription(
+        key="charge_state_charging_state",
```

Comment:
From the `strings.json` it looks like you know all possible values of this sensor.
Can we set the device class to `SensorDeviceClass.ENUM` and specify the options?

Reply:
Oh yes, forgot about Enum types. I'll do that.

Reply:
Done.